app/views/games/categories/show.slim

```diff
@@ -1,20 +1,54 @@
 - title @game.name
 - content_for(:header) do
-  ol.breadcrumb
+  ol.breadcrumb.shadow
+    li.breadcrumb-item = link_to "Splits I/O", root_path
```

Comment:
A small thing with all breadcrumbs, there is a larger gap on the right side of each breadcrumb when there is more than one.  This is caused by a left padding on each subsequent breadcrumb after the first one.  It's uniform on the live site and not sure if you want this to stay the same.